### ISSUE TITLE
The function _make_error_response is not visible

### ISSUE THREAD

**Issue by oklas** [Maki-Daisuke/p5-JSON-RPC2-AnyEvent#2]
I am using for long time this module on different systems and perl versions. Suddenly error appears at centos with perl (v5.16.3) built for x86_64-linux-thread-multi from [here](https://github.com/Maki-Daisuke/p5-JSON-RPC2-AnyEvent/blob/master/lib/JSON/RPC2/AnyEvent/Server/Handle.pm#L52), the error is:

> Undefined subroutine &JSON::RPC2::AnyEvent::_make_error_response called at /home/larix/host-main/nginx-service/local/lib/perl5/JSON/RPC2/AnyEvent/Server/Handle.pm line 52.

**Comment by oklas** [comment]
Error seems has a reason. Actually function `_make_error_response` is defined at `JSON::RPC2::AnyEvent::Server` but not at `JSON::RPC2::AnyEvent` (however it present in `use` statement).

**Comment by Maki-Daisuke** [comment]
@oklas, I run tests on CentOS with perl-5.16.3, but it worked with no error:

```
[root@a97828876bc0 app]# cat /etc/redhat-release
CentOS Linux release 8.1.1911 (Core)
[root@a97828876bc0 app]# perl -v

This is perl 5, version 16, subversion 3 (v5.16.3) built for x86_64-linux
(with 1 registered patch, see perl -V for more detail)

Copyright 1987-2012, Larry Wall

Perl may be copied only under the terms of either the Artistic License or the
GNU General Public License, which may be found in the Perl 5 source kit.

Complete documentation for Perl, including FAQ lists, should be found on
this system using "man perl" or "perldoc perl".  If you have access to the
Internet, point your browser at http://www.perl.org/, the Perl Home Page.

[root@a97828876bc0 app]# prove -I./lib/ ./t
t/00_compile.t ..... ok
t/01_sync.t ........ ok
t/02_async.t ....... ok
t/03_tcp_server.t .. ok
t/04_argspec.t ..... ok
All tests successful.
Files=5, Tests=62,  6 wallclock secs ( 0.09 usr  0.03 sys +  0.49 cusr  0.16 csys =  0.77 CPU)
Result: PASS
```

Can you tell me your CentOS version and test code causing the error?

**Comment by Maki-Daisuke** [comment]
@oklas, oh, you are right. `JSON::RPC2::AnyEvent::_make_error_response` should be `JSON::RPC2::AnyEvent::Server::_make_error_response`!
And, the error happens only when the underlying I/O, network I/O in most cases, causes error. That's why my test worked with no error.

I'll fix and ship it soon.

Thank you!

**Comment by Maki-Daisuke** [comment]
Released v0.03
